index.html

```diff
@@ -977,6 +982,10 @@ <h2>
               say, an item at the end of the list always takes precedence over
               any item at the beginning of the list (see example below).
             </p>
+            <p>
+              Decisions on behalf of the user (e.g., steps 6 and 12) MUST NOT
+              restrict competition.
```

Comment:
I understand and support the intent of this change. However, I don't think a normative requirement around competition is the way to go because such a requirement would be fundamentally untestable.
My counter proposal would be to include a pointer to the [W3C Antitrust and Competition Guidance](https://www.w3.org/Consortium/Legal/2017/antitrust-guidance) in the "Status of this document" section at the beginning of the spec, an informative note directly above Example 7 in the spec text (so replacing your change at line 985) explicitly noting that the algorithm has the potential to be used in an anti-competitive way , and a reference pointer in D.1 Normative references to the W3C Antitrust and Competition Guidance policy (which I would justify as a normative reference due to the "MUST" statement here -> " Participants must ensure that their conduct does not violate antitrust and competition laws and regulations. "

Reply:
Agree.

Reply:
Thanks for the proposal, I agree.

Reply:
Based on further discussion within the staff I have removed this piece of the pull request.